etc/builtin.conf

```diff
@@ -84,8 +84,6 @@ osd-outline-size=0
 osd-shadow-offset=4
 script-opt=console-border_size=0
 script-opt=stats-border_size=0
-script-opt=stats-shadow_x_offset=4
-script-opt=stats-shadow_y_offset=4
```

Comment:
Shouldn't profile still override, in case user wants to have different one than default in non-box mode? Or it is on user to extend profile in this case?

Reply:
Dunno but if it should then this should also reset blur.

Reply:
Actually the advantage of removing is that if you want to apply a different shadow offset on top of this profile, it will update automatically in stats.

Reply:
I don't know. Anyway it is consistent with everything else we do recently, so let's do it.